src/main/java/com/cburch/logisim/fpga/hdlgenerator/ToplevelHDLGeneratorFactory.java

```diff
@@ -279,45 +274,42 @@ public String GetSubDir() {
       }
     }
     if (NrOfInputPorts > 0) {
```

Comment:
Can we have `NrOfInputPorts` renamed as side-effect?

Reply:
See my remark above